cmake/macos/compilerconfig.cmake

```diff
@@ -21,8 +21,8 @@ endif()
 set_property(CACHE CMAKE_OSX_ARCHITECTURES PROPERTY STRINGS arm64 x86_64)
 
 # Ensure recent enough Xcode and platform SDK
-set(_obs_macos_minimum_sdk 14.2) # Keep in sync with Xcode
-set(_obs_macos_minimum_xcode 15.1) # Keep in sync with SDK
+set(_obs_macos_minimum_sdk 15.0) # Keep in sync with SDK
+set(_obs_macos_minimum_xcode 16.0) # Keep in sync with Xcode
```

Comment:
The SDK/Xcode is not a typo. They’re meant to say that these two lines should be kept in sync with each other, i.e. for the Xcode version that it should be kept in sync with the SDK version shipped with that Xcode version, and vice versa.

Reply:
I see, seemed a bit weird to me.

Reply:
Updated, I'll also drop the first commit if #11624 gets merged beforehand.